fastify.js

```diff
@@ -566,17 +566,21 @@ function fastify (options) {
   function addHook (name, fn) {
     throwIfAlreadyStarted('Cannot call "addHook" when fastify instance is already started!')
 
+    if (fn == null) {
```

Comment:
Only expect function.

```suggestion
    if (typeof fn !== 'function') {
```

Reply:
Let's open a `good first issue` for that wdyt?